Sources/TokamakCore/Tokens/Color.swift

```diff
@@ -252,6 +268,16 @@ public extension Color {
   }
 }
 
+extension Color: CustomStringConvertible {
+  public var description: String {
+    if let providerDescription = provider as? CustomStringConvertible {
+      return providerDescription.description
+    } else {
+      return String(describing: self)
```

Comment:
Just noticed that this will recurse if `provider` doesn’t conform to `CustomStringConvertible`. Should this be `String(describing: provider)`, or should we require all providers conform to `CustomStringConvertible`?

Reply:
Thanks for pointing it out! @carson-katri WDYT?

Reply:
Good catch! I think `String(describing: provider)` just in case, but also implement `CustomStringConvertible` on all the providers since they really are only made in TokamakCore, if that makes sense?